sdk/identity/identity/src/util/subscriptionUtils.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CredentialLogger, formatError } from "./logging";
+
+/**
+ * @internal
+ */
+export function checkSubscription(logger: CredentialLogger, subscription: string): void {
+  if (!subscription.match(/^[0-9a-zA-Z-. ]+$/)) {
+    const error = new Error(
+      "Invalid subscription provided. You can locate your subscription by following the instructions listed here: https://learn.microsoft.com/entra/fundamentals/how-to-find-tenant.",
```

Comment:
The link provided seems geared more towards finding the tenant ID. Perhaps this one might be better: https://learn.microsoft.com/azure/azure-portal/get-subscription-tenant-id ?

Reply:
Thank you for the link, updated with the feedback!